example/lib/line_chart/samples/line_chart_sample_betweenbarsdata.dart

```diff
@@ -0,0 +1,146 @@
+import 'package:fl_chart/fl_chart.dart';
+import 'package:flutter/material.dart';
+
+class LineChartSampleBetweenBarsData extends StatelessWidget {
```

Comment:
Please follow our namings, (line_chart_sample{x}.dart), x is a number, we should follow it to prevent making inconsistencies (we use this convention for showing samples in documentations, and we name the images that have been saved in `./repo_files/images/line_chart/line_chart_sample_{x}.png`),
Please rename it to `line_chart_sample7.dart` and put it in the last sample position (we add samples sequentially)